### PR TITLE
[M678] Send out emails on copied projects

### PR DESCRIPTION
**Pull request by gridcell**

### CODE REVIEW

**Comment by gridcell** [comment]
[Email user's that are included in a copied project](https://trello.com/c/rTFq2mix/678-email-users-that-are-included-in-a-copied-project)

**Comment by gridcell** [comment]
@saanobhaai Should code review https://github.com/data-mermaid/mermaid-api/pull/253 first and then I'll update this branch with dev

**Review by saanobhaai — APPROVED** [review]

**Comment by saanobhaai** [comment]
I like the move to `bulk_create` and this looks good to go after the other PR is merged.

**Pull request merged**
